cpp/tests/test_ide_secir.cpp

```diff
@@ -158,6 +158,57 @@ TEST_F(ModelTestIdeSecir, compareWithPreviousRunTransitions)
     }
 }
 
+// Check that the start time of the simulation is determined by the given time points for the transitions.
+TEST(IdeSecir, checkStartTime)
+{
+    using Vec = mio::TimeSeries<ScalarType>::Vector;
+
+    ScalarType tmax   = 3.0;
+    ScalarType N      = 10000.;
+    ScalarType deaths = 10.;
+    ScalarType dt     = 1.;
+    ScalarType t0     = 2.;
+
+    int num_transitions = (int)mio::isecir::InfectionTransition::Count;
+
+    // Create TimeSeries with num_transitions elements where transitions needed for simulation will be stored.
+    mio::TimeSeries<ScalarType> init(num_transitions);
+
+    // Define transitions that will be used for initialization.
+    Vec vec_init(num_transitions);
+    vec_init[(int)mio::isecir::InfectionTransition::SusceptibleToExposed]                 = 1.0;
+    vec_init[(int)mio::isecir::InfectionTransition::ExposedToInfectedNoSymptoms]          = 0.0;
```

Comment:
```suggestion
Vec vec_init                                                          = Vec::Constant(num_transitions, 0.);
    vec_init[(int)mio::isecir::InfectionTransition::SusceptibleToExposed] = 1.0;
    vec_init[(int)mio::isecir::InfectionTransition::InfectedNoSymptomsToInfectedSymptoms] = 8.0;
```

This would be shorter and clearer. Take what you prefer. The suggestion should replace lines 178-188.